config/sudoers

```diff
@@ -31,6 +31,7 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/create-cert
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/sign-message
```

Comment:
As noted in https://github.com/votingworks/vxsuite/pull/3518#discussion_r1207191880, I've landed on the same control flow for message signing as I did for cert creation.

So you don't have to dig too much, the core logic for using the TPM to sign messages lives [here](https://github.com/votingworks/vxsuite/blob/79713f11a991cce1708c03237f65fcc382b6ec14/libs/auth/src/openssl.ts#L466-L495).